src/components/TagMenu/index.web.tsx

```diff
@@ -49,8 +49,8 @@ export function TagMenu({
       {
         label: _(msg`See ${truncatedTag} posts`),
         onPress() {
-          navigation.navigate('Search', {
-            q: tag,
+          navigation.push('Hashtag', {
+            tag: tag.slice(1).replaceAll('#', '%23'),
```

Comment:
Taking the tag and just removing the first character. When we press this button, we know it's going to have at least one # which we'll remove.

The remaining '#' should all be encoded.